website/versioned_docs/version-0.4.x/set-up-your-environment.mdx

```diff
@@ -484,14 +501,142 @@ IMPORTANT: Ceramic API running on 0.0.0.0:7007
+</details>
+<details>
+  <summary>Where in the system do I run all of the commands?</summary>
+  <div>
+    <div>
+      Sometimes, especially when using JavaScript package managers to install Ceramic and ComposeDB,
+      it's easy to forget that you need to run all of the commands in the app's directory. With
+      JavaScript package managers you need to open 2-3 terminal windows and run different commands,
+      so it's easy to miss that you can be in a wrong directory. Please make sure you run all the
+      commands where they're supposed to run.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>Where can I find a Ceramic node configuration file, daemon.config.json?</summary>
+  <div>
+    <div>
+      When installing ComposeDB with JavaScript package managers, at some point you need to edit
+      your Ceramic node config file. By default, it's in your home directory, in .ceramic folder
+      (*not* in the app directory). It's easy to miss this details so please check the path. This
+      command should take you to the right directory: cd ~/.ceramic
+    </div>
+  </div>
+</details>
+<details>
+  <summary>How to restart a node after stopping it?</summary>
+  <div>
+    When you use wheel to install Ceramic and ComposeDB, it takes care of the whole installation
+    process. But please note that wheel is just an installer, not a node launcher. If you want to
+    launch Ceramic and ComposeDB again, after you have stopped it, you need to launch Ceramic daemon
+    again and then launch ComposeDB.
+  </div>
+  <div>
+    You can launch Ceramic daemon by running the following command: ceramic daeomn
+    --network=InMemory
+  </div>
+  <div>You can launch ComposeDB by running the command: composedb</div>
+  <div>
+    More on all of the composedb command options can be found in "2. Create your composite" section
+    of this Getting Started guide.
+  </div>
+</details>
+<details>
+  <summary>How do I interact with the data once the node is running?</summary>
+  <div>
+    <div>
+      You interact with data through a GraphQL Server. You can find all the details on how to set it
+      up, launch, and interact with your data in section of this guide, "3. Interact with data"
+    </div>
+  </div>
+</details>
+<details>
+  <summary>
+    Error when creating a composite: ✖ request to http://localhost:7007/(...) failed, reason:
+    connect ECONNREFUSED ::1:7007
+  </summary>
+  <div>
+    <div>The most likely cause is using Node.js v18. Please try using Node.js v16.</div>
+  </div>
+</details>
+<details>
+  <summary>Error: npm ERR! code EACCESS</summary>
+  <div>
+    <div>
+      The most likely cause is read/write access on your system. Try running the command with sudo
+      privileges.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>What if my question not answered on this page?</summary>
+  <div>
+    <div></div>
+    <div>
+      If your question is not answered in this guide, we recommend visiting our Community Forum or
```

Comment:
Would suggest adding a direct link to the forum. And for technical questions it's best to send people to the forum instead of Discord so that we can keep track of their questions.

Reply:
Same as one of the answers above: with our current Docusaurus config, we can't add links in the FAQ elements.
Updated to go to forum.